docs/json-codec-backend/codec-docs.md

```diff
@@ -0,0 +1,67 @@
+## Json Codec Backend
+The purpose of this documentation is to give a explanation of how the Json Codec Backend works and how to use it.
+The Json Codec backend is a tool that generates encoders and decoders from a Morphir IR using the Circe JSON library.
```

Comment:
I think we need a bit more explanation here. A few things to mention:
- This is not a standalone backend. It's rather a feature on the Scala backend.
- What encoders and decoders are might not be obvious for everyone. Let's explicitly state that this helps you read and write the Scala types that the Scala backend generates from/to JSON.